### ISSUE TITLE
The drop down menu pushes the 'Make remote work' title down.

### ISSUE THREAD

**Issue by seanred360** [VANIMEHTA/csb-ygs4zj#3]
You should absolutely position the dropdown so that it does not change your pages layout.

**Comment by VANIMEHTA** [comment]
> 

Yeah I faced that issue but couldn’t fix it, can u suggest changes for same?

**Comment by seanred360** [comment]
You need to add position absolute to the #hidden and #hidden1. This removes these elements from the document flow, which means they cannot affect other elements. Look at this pull request https://github.com/VANIMEHTA/csb-ygs4zj/pull/8/commits/af3a8e895f52ec669176c364ccd08c486819a3c8
